api/answer.py

```
@@ -355,6 +424,110 @@ def load_token(self):
     def _init_tiku(self):
         self.load_token()
 
+class TikuGo(Tiku):
+    # GO题（网课小工具题库）实现
+    def __init__(self) -> None:
+        super().__init__()
+        self.name = 'GO题（网课小工具题库）'
```

Ruff (0.15.7)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)